From 679ab5de38082437484902cf644dbc980453e392 Mon Sep 17 00:00:00 2001
From: Rajaganesh Rathinasabapathi <Rajaganesh.Rathinasabapathi@amd.com>
Date: Thu, 4 Aug 2022 00:27:12 -0500
Subject: [PATCH] ARM:dts:Aspeed: remove ununsed i3c buses from SP6

removed ununsed i3c buses from Sp6 platforms
enabled imx3112 mux device for apml buses

Signed-off-by: Rajaganesh Rathinasabapathi <Rajaganesh.Rathinasabapathi@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-cinnabar.dts |  9 ++------
 arch/arm/boot/dts/aspeed-bmc-amd-shale64.dts  | 21 ++-----------------
 arch/arm/boot/dts/aspeed-bmc-amd-shale96.dts  | 21 ++-----------------
 arch/arm/boot/dts/aspeed-bmc-amd-sunstone.dts |  9 ++------
 4 files changed, 8 insertions(+), 52 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-cinnabar.dts b/arch/arm/boot/dts/aspeed-bmc-amd-cinnabar.dts
index 511c58076527..3a43c85e3a48 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-cinnabar.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-cinnabar.dts
@@ -696,12 +696,12 @@
 	bus_id = <0x04>;

 	/* Mux Bypassed by default on SP6, enable if added later */
-#ifdef BYPASSED_MUX
+//#ifdef BYPASSED_MUX
 	imx3112_p0: i3cmux@70,4CC00000000 {
 		reg = <0x70 0x4CC 0x00000000>;
 		assigned-address = <0x70>;
 	};
-#endif
+//#endif

 	sbtsi_p0_0: sbtsi@4c,22400000001 {
 		reg = <0x4c 0x224 0x00000001>;
@@ -713,11 +713,6 @@
 	};
 };

-&i3c5 {
-	// NC
-	status = "okay";
-};
-
 #ifdef ENABLE_I2C_APML
 &i2c2 {
 	// P0 APML - i2c10
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-shale64.dts b/arch/arm/boot/dts/aspeed-bmc-amd-shale64.dts
index e56d86996087..afdc6d705341 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-shale64.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-shale64.dts
@@ -684,18 +684,6 @@
 		reg = <0x5a 0x2C0 0x00000002>;
 		assigned-address = <0x5a>;
 	};
- };
-
-&i3c2 {
-	// NC
-	status = "okay";
-
-};
-
-&i3c3 {
-	// NC
-	status = "okay";
-
 };

 &i3c4 {
@@ -707,12 +695,12 @@
 	bus_id = <0x04>;

 	/* Mux Bypassed by default on SP6, enable if added later */
-#ifdef BYPASSED_MUX
+//#ifdef BYPASSED_MUX
 	imx3112_p0: i3cmux@70,4CC00000000 {
 		reg = <0x70 0x4CC 0x00000000>;
 		assigned-address = <0x70>;
 	};
-#endif
+//#endif

 	sbtsi_p0_0: sbtsi@4c,22400000001 {
 		reg = <0x4c 0x224 0x00000001>;
@@ -724,11 +712,6 @@
 	};
 };

-&i3c5 {
-	// NC
-	status = "okay";
-};
-
 #ifdef ENABLE_I2C_APML
 &i2c2 {
 	// P0 APML - i2c10
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-shale96.dts b/arch/arm/boot/dts/aspeed-bmc-amd-shale96.dts
index 5bacb63bb28f..e547f4d6674c 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-shale96.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-shale96.dts
@@ -684,18 +684,6 @@
 		reg = <0x5a 0x2C0 0x00000002>;
 		assigned-address = <0x5a>;
 	};
- };
-
-&i3c2 {
-	// NC
-	status = "okay";
-
-};
-
-&i3c3 {
-	// NC
-	status = "okay";
-
 };

 &i3c4 {
@@ -707,12 +695,12 @@
 	bus_id = <0x04>;

 	/* Mux Bypassed by default on SP6, enable if added later */
-#ifdef BYPASSED_MUX
+//#ifdef BYPASSED_MUX
 	imx3112_p0: i3cmux@70,4CC00000000 {
 		reg = <0x70 0x4CC 0x00000000>;
 		assigned-address = <0x70>;
 	};
-#endif
+//#endif

 	sbtsi_p0_0: sbtsi@4c,22400000001 {
 		reg = <0x4c 0x224 0x00000001>;
@@ -724,11 +712,6 @@
 	};
 };

-&i3c5 {
-	// NC
-	status = "okay";
-};
-
 #ifdef ENABLE_I2C_APML
 &i2c2 {
 	// P0 APML - i2c10
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-sunstone.dts b/arch/arm/boot/dts/aspeed-bmc-amd-sunstone.dts
index d482c0a3e25a..c44f60a6f750 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-sunstone.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-sunstone.dts
@@ -792,12 +792,12 @@
 	bus_id = <0x04>;
 	set_dasa;
 	/* Mux Bypassed by default on SP6, enable if added later */
-#ifdef BYPASSED_MUX
+//#ifdef BYPASSED_MUX
 	imx3112_p0: i3cmux@70,4CC00000000 {
 		reg = <0x70 0x4CC 0x00000000>;
 		assigned-address = <0x70>;
 	};
-#endif
+//#endif

 	sbtsi_p0_0: sbtsi@4c,22400000001 {
 		reg = <0x4c 0x224 0x00000001>;
@@ -809,11 +809,6 @@
 	};
 };

-&i3c5 {
-	// NC
-	status = "okay";
-};
-
 #ifdef ENABLE_I2C_APML
 &i2c2 {
 	// P0 APML - i2c10
--
2.17.1

